pkg/cli/admin/inspect/inspect.go

```diff
@@ -61,8 +62,10 @@ type InspectOptions struct {
 
 	fileWriter    *MultiSourceFileWriter
 	builder       *resource.Builder
+	sincetime     time.Duration
 	args          []string
 	namespace     string
+	since         int64
```

Comment:
This has to be string, since `oc log` can accept values such as `1h`, etc that are then parsed using `ParseRFC3339` method.

Reply:
in podlogoptions there is field SinceTime and it has be in int64 https://github.com/openshift/oc/blob/f2b01c4e4ae8c4ca11caabf8cb8e76b7a28b7009/vendor/k8s.io/api/core/v1/types.go#L4862 
it cannot be string.  can you explain how it can be done

Reply:
@soltysh ^

Reply:
Check `oc logs` command, please. For this particular one: https://github.com/openshift/oc/blob/6b061e3427d32b58113a7f1cae28f4c7f89ec71a/vendor/k8s.io/kubectl/pkg/cmd/logs/logs.go#L190-L195